### ISSUE TITLE
why accuracy is diffrent when use subvision=64,batch=64 and subvision=32,batch=64?

### ISSUE THREAD

**Issue by yxwLearn** [AlexeyAB/darknet#2890]
@AlexeyAB  

Excuse me
when i  trainning with bacth=64,subvision=32,the  accuracy curve is more than 47%, and improve soon,but trainning whith bacth=64,subvision=64,the  accuracy curve is less than 20%, and improve very slowly, even declined.

**Comment by AlexeyAB** [comment]
@yxwLearn 

Check again, that you do 100% the same, the only difference is subdivisions.

Can you show screenshots of Loss & mAP chart for both cases?

**Comment by yxwLearn** [comment]
I checked again,only difference is subdivisions.now i use subvision =32,at 2650 iteration the accuracy is 70%,but use subvision = 64,at 2650 iteration the accuracy is 15% .

**Comment by AlexeyAB** [comment]
Do you use one cfg file or two cfg-files?
What dataset do you use?

**Comment by yxwLearn** [comment]
I used one cfg file。I  label each object  on images for yolov3 trainning。the dataset is many fruit。

**Comment by AlexeyAB** [comment]
Can you show screenshots of Loss & mAP chart for both cases?

**Comment by yxwLearn** [comment]
@AlexeyAB 
I come from China，maybe the firewall stop me upload file,so there something  goes
 wrong when i upload the screenshots.

**Comment by AlexeyAB** [comment]
I have met the difference in accuracy of a maximum of 1-2 mAP%, but I never met 2x mAP drop between subvision=32 / 64.

Batch normalization can be very sensitive to `mini_batch_size = batch / subdivisions`

Did you change anything in cfg-file?
Do your images have a big difference in lighting/exposure?

**Comment by PhilipWu526** [comment]
Hello, @yxwLearn .
I am also curious about the problems raised by AlexeyAB .
How is the result?

**Comment by keineahnung2345** [comment]
In one experiment, I set batch size as 1 and subdivisions as 1, and map is 91.95 %; 
in another experiment, I set batch size as 8 and subdivisions as 1, and map becomes 84.68 %.
Why does this happen?

**Comment by AlexeyAB** [comment]
@keineahnung2345 Do you use batch size as 1 and subdivisions as 1 for training or for detection?

**Comment by keineahnung2345** [comment]
I use batch=64 and subdivisions=16 in training, and trains for 50000 iterations.
I use the same weights for testing.

**Comment by keineahnung2345** [comment]
@AlexeyAB Sorry but I later found the code I previously used was a modified version(not originally from this repo), and when I tried with the original version, everything works as expected.
